lib/faexport/scraper.rb

```diff
@@ -234,7 +236,12 @@ def budlist(name, page, is_watchers)
     mode = is_watchers ? 'watched_by' : 'watches'
     html = fetch("user/#{escape(name)}")
     id = find_id(html)
-    html = fetch("budslist/?name=#{escape(name)}&uid=#{id}&mode=#{mode}&page=#{page}")
+    if mode == 'watched_by'
```

Comment:
You should be able to just change `mode` to store `'to'` and `'by'`, it's not being used for anything else.